src/main/webapp/app/shared/course-management-tab-bar/course-management-tab-bar.component.ts

```diff
@@ -0,0 +1,121 @@
+import { Component, Input, OnDestroy, OnInit } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { HttpErrorResponse } from '@angular/common/http';
+import { Subject, Subscription } from 'rxjs';
+import { Course, isCommunicationEnabled } from 'app/entities/course.model';
+import { CourseManagementService } from 'app/course/manage/course-management.service';
+import { ButtonSize } from 'app/shared/components/button.component';
+import { EventManager } from 'app/core/util/event-manager.service';
+import {
+    faChartBar,
+    faClipboard,
+    faComments,
+    faEye,
+    faFilePdf,
+    faFlag,
+    faGraduationCap,
+    faHeartBroken,
+    faListAlt,
+    faPersonChalkboard,
+    faTable,
+    faTimes,
+    faUserCheck,
+    faWrench,
+} from '@fortawesome/free-solid-svg-icons';
+import { FeatureToggle } from 'app/shared/feature-toggle/feature-toggle.service';
+import { CourseAdminService } from 'app/course/manage/course-admin.service';
+
+@Component({
+    selector: 'jhi-course-management-tab-bar',
+    templateUrl: './course-management-tab-bar.component.html',
+    styleUrls: ['./course-management-tab-bar.component.scss'],
+})
+export class CourseManagementTabBarComponent implements OnInit, OnDestroy {
+    readonly FeatureToggle = FeatureToggle;
+    readonly ButtonSize = ButtonSize;
+
+    course?: Course;
+
+    @Input() showCourseEditButtons = false;
```

Comment:
Where is this input used? It seems unsued if I don't miss anything.

Reply:
Hi, you are actually right. This was used when the course-management-tab-bar was inserted directly. Now it is not used anymore. I'll fix that.
Thank you for noticing